modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryMetricsAdapter.java

```diff
@@ -126,41 +125,106 @@ public void update(long duration, boolean fail) {
      *
```

Comment:
It seems to me that javadoc became obsolete and must be fixed. Isn't it?

Reply:
fixed.